tutorials/vue/vue-inject.md

```diff
@@ -0,0 +1,52 @@
+Vue's provide/inject API is a neat way to send props directly
+to the component you want while avoiding prop drilling.
+What you do is you declare what variables you will send, i.e. provide,
+in the setup property and then in the component that needs them you use the inject property.
+Here is an example below:
+
+<div id = "content"></div>
+<script src="https://unpkg.com/vue@next"></script>
+<script>
+const app = Vue.createApp({
+  setup: function() {
+      const state = Vue.reactive({message: 'Hi grandma!'});
+      Vue.provide('state', state);
+      return state;
+  },
+  template: `<div><child /></div>`
+});
+app.component('child', {
+    template: `<grandchild />`
+});
+app.component('grandchild', {
+    inject: ['state'],
+    template: `<h1>{{state.message}}</h1>`
+});
+app.mount('#content');
+</script>
+
+```html
+<div id = "content"></div>
+<script src="https://unpkg.com/vue@next"></script>
+<script>
+const app = Vue.createApp({
+  setup: function() {
+      const state = Vue.reactive({message: 'Hi grandma!'});
+      Vue.provide('state', state);
+      return state;
+  },
+  template: `<div><child /></div>`
+});
+app.component('child', {
+    template: `<grandchild />`
+});
+app.component('grandchild', {
+    inject: ['state'],
+    template: `<h1>{{state.message}}</h1>`
+});
+app.mount('#content');
+</script>
+```
+As you can see, we avoided having to put the state property
```

Comment:
Use 2nd person "you" rather than "we", and no need for "as you can see." More like "With `inject`, you don't have to explicitly pass a `state` prop to the child component."